app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -73,6 +74,10 @@
           }
         }
       }
+      if (/^\/digital-outcomes-and-specialists(.*?)\/opportunities\/(\d+)$/.test(pathname)) {
```

Comment:
Great shout, it was actually formatted completely differently. 
Especially important given the code style presentation im giving on Monday 😬